### DISCUSSION TITLE
issue with ffmpeg 5 and almalinux 8.7

### DISCUSSION THREAD

**Discussion by wezo3** [savonet/liquidsoap#2743] [Installation] [answered]
**Describe the bug**
there issue happened after use almalinux 8.7 its install ffmpeg version 5.0 and this appear error liquidsoap

liquidsoap --version
liquidsoap: error while loading shared libraries: libswscale.so.5: cannot open shared object file: No such file or directory
I try to make sym link and not works and same issue with this libs
ln -s /usr/lib64/libswscale.so.6.4.100 /usr/lib64/libswscale.so.5
ln -s /usr/lib64/libswresample.so.4.3.100 /usr/lib64/libswresample.so.3
ln -s /usr/lib64/libavfilter.so.8.24.100 /usr/lib64/libavfilter.so.7
ln -s /usr/lib64/libavdevice.so.59.4.100 /usr/lib64/libavdevice.so.58
ln -s /usr/lib64/libavutil.so.57.17.100 /usr/lib64/libavutil.so.56
ln -s /usr/lib64/libavformat.so.59.16.100 /usr/lib64/libavformat.so.58
ln -s /usr/lib64/libavcodec.so.59.18.100 /usr/lib64/libavcodec.so.58

liquidsoap --version
liquidsoap: /lib64/libswscale.so.5: version `LIBSWSCALE_5' not found (required by liquidsoap)
liquidsoap: /lib64/libavdevice.so.58: version `LIBAVDEVICE_58' not found (required by liquidsoap)
liquidsoap: /lib64/libswresample.so.3: version `LIBSWRESAMPLE_3' not found (required by liquidsoap)
liquidsoap: /lib64/libavformat.so.58: version `LIBAVFORMAT_58' not found (required by liquidsoap)
liquidsoap: /lib64/libavcodec.so.58: version `LIBAVCODEC_58' not found (required by liquidsoap)
liquidsoap: /lib64/libavfilter.so.7: version `LIBAVFILTER_7' not found (required by liquidsoap)
liquidsoap: /lib64/libavutil.so.56: version `LIBAVUTIL_56' not found (required by liquidsoap)

and we can not install ffmpeg v4 by yum 

**Version details**

- OS: almalinux 8.7
- liquidsoap.2.1.2

**Install method**
by opam

its was working good on almalinux 8.6 when was install ffmpeg v4
can you solve this to compatible with ffmpeg v5 ?

Regards

**Comment by cdgraff** [comment, marked as answer]
Try doing:
`opam reinstall ffmpeg-avutil`

The way ocaml bindings work, they remember the location of the libraries they're linked against you need to rebuild them when you upgrade/change that library's location.

**Reply by wezo3** [reply]
thank you, I will test this
